etn-ccis/blui-react-workflows
# Bug/4508 Add change password in usermenu dropdown at homepage

Fixes # added change password in the user menu at homepage.



#### Changes proposed in this Pull Request:

-
-
-



#### Screenshots / Screen Recording (if applicable)

-
<img width="717" alt="Screenshot 2023-08-04 at 11 35 51 PM" src="https://github.com/etn-ccis/blui-react-workflows/assets/133086327/a6627803-688f-45c9-a19c-fd061a3d0e21">





#### To Test:

- login to homepage
- select change password in the user menu dropdown
- able to see the change password dialog box



#### Any specific feedback you are looking for?

-
